packages/admin/admin/src/common/toolbar/automatictitleitem/ToolbarAutomaticTitleItem.tsx

```diff
@@ -9,28 +9,41 @@ export interface ToolbarAutomaticTitleItemProps {
     typographyProps?: TypographyTypeMap["props"];
 }
 
-function AutomaticTitleItem({ typographyProps }: ToolbarAutomaticTitleItemProps): React.ReactElement {
+export type ToolbarAutomaticTitleItemClassKey = "root" | "typography";
+
+const styles = () => {
+    return createStyles<ToolbarAutomaticTitleItemClassKey, ToolbarAutomaticTitleItemProps>({
+        root: {},
+        typography: {},
+    });
+};
+
+function AutomaticTitleItem({ typographyProps, classes }: ToolbarAutomaticTitleItemProps & WithStyles<typeof styles>): React.ReactElement {
     const stackApi = useStackApi();
 
     return (
-        <ToolbarItem>
-            <Typography variant="h4" {...typographyProps}>
+        <ToolbarItem classes={{ root: classes.root }}>
+            <Typography variant="h4" classes={{ root: classes.typography }} {...typographyProps}>
```

Comment:
`typographyProps` can also include `classes`, this would override value `classes.typography`. 
Can you merge those together? Maybe like this:

```tsx
const { classes: typographyClasses = {}, ...restTypographyProps } = typographyProps;
typographyClasses.root = clsx(typographyClasses.root, classes.typography);
// ...
<Typography variant="h4" classes={typographyClasses} {...restTypographyProps}>

Reply:
I used [ToolbarTitleItem](https://github.com/vivid-planet/comet/blob/main/packages/admin/admin/src/common/toolbar/titleitem/ToolbarTitleItem.tsx) as a reference. Also tried your suggestion now but got this: 
![Screenshot 2023-05-11 at 08 49 33](https://github.com/vivid-planet/comet/assets/72875628/11faaf73-fdc2-46f5-811b-eaaff86d5be5)

Reply:
We probably forgot to consider this in most components, including `ToolbarTitleItem`, but I think we should always do this incase we want to add classes to those components when using them in a project. 

I think the issue is, that `typographyProps` can be `undefined`.  
It should work, if you give it a default value of `{}` in the components props: 

```diff
- function AutomaticTitleItem({ typographyProps, classes }: ToolbarAutomaticTitleItemProps & WithStyles<typeof styles>): React.ReactElement {
+ function AutomaticTitleItem({ typographyProps = {}, classes }: ToolbarAutomaticTitleItemProps & WithStyles<typeof styles>): React.ReactElement {
```

Reply:
got it, thank you!